engrave-faces.el

```diff
@@ -287,6 +291,17 @@ This function is lifted from htmlize."
        (font-lock-regexp-grouping-backslash :short "fl-regexp-backslash" :slug "rb"                        :weight bold)
        ;; org-faces.el
        (org-block                           :short "org-block"           :slug "ob") ; forcing no background is preferable
+       (org-block-begin-line                :short "org-block-begin"     :slug "obb") ; forcing no background is preferable
+       (org-block-end-line                  :short "org-block-end"       :slug "obe") ; forcing no background is preferable
+       ;; outlines
+       (outline-1 :foreground blue                        :short "o-1")
+       (outline-2 :foreground magenta                     :short "o-2")
+       (outline-3 :foreground violet                      :short "o-3")
+       (outline-4 :foreground (doom-lighten blue 0.25)    :short "o-4")
+       (outline-5 :foreground (doom-lighten magenta 0.25) :short "o-5")
+       (outline-6 :foreground (doom-lighten blue 0.5)     :short "o-6")
+       (outline-7 :foreground (doom-lighten magenta 0.5)  :short "o-7")
+       (outline-8 :foreground (doom-lighten blue 0.8)     :short "o-8")
```

Comment:
I'm not planning on adding a doom colour dependancy.

Reply:
I'd also like it if we had outline/slug like `outline-1`/`oa`.

Reply:
> I'm not planning on adding a doom colour dependancy.

That's on me, I meant to evaluate it to get the proper colour code before I pushed

> I'd also like it if we had outline/slug like outline-1/oa.

I did add them but it seems they were lost in another commit, oops

Reply:
> That's on me, I meant to evaluate it to get the proper colour code before I pushed

Note that the default style is supposed to be the values using the `default` theme.

Reply:
> > That's on me, I meant to evaluate it to get the proper colour code before I pushed
> 
> Note that the default style is supposed to be the values using the `default` theme.

I know, I've been evaluating in vanilla emacs for the most part